<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>creditcalc</class>
 <widget class="QMainWindow" name="creditcalc">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>587</width>
    <height>658</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>creditcalc</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="push_calc">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>330</y>
      <width>441</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>РАССЧИТАТЬ</string>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>50</y>
      <width>503</width>
      <height>251</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="3" column="0">
      <widget class="QLabel" name="type_label">
       <property name="text">
        <string>Тип ежемесячных платежей</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="time_label">
       <property name="text">
        <string>Срок кредита</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QRadioButton" name="type_an">
         <property name="text">
          <string>Аннуитентный</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="type_diff">
         <property name="text">
          <string>Дифференцированный</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="proz_label">
       <property name="text">
        <string>Процентная ставка</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="sum_label">
       <property name="text">
        <string>Сумма кредита</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLineEdit" name="time_year_edit">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="time_year_line">
         <property name="text">
          <string>лет</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="time_month_edit">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="time_month_line">
         <property name="text">
          <string>месяцев</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLineEdit" name="sum_edit">
         <property name="text">
          <string>0.0</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="rub_line">
         <property name="text">
          <string>руб.</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLineEdit" name="proz_edit">
         <property name="text">
          <string>0.0</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="proz_line">
         <property name="text">
          <string> %   </string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>430</y>
      <width>471</width>
      <height>171</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="1">
      <widget class="QLineEdit" name="every_month_edit">
       <property name="text">
        <string>0.0</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="result_edit">
       <property name="text">
        <string>0.0</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="every_month_label">
       <property name="text">
        <string> Ежемесячный платеж:          </string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="overpayment_edit">
       <property name="text">
        <string>0.0</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="result_label">
       <property name="text">
        <string> Общая выплата:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="overpayment_label">
       <property name="text">
        <string> Переплата по кредиту:</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionCredit">
   <property name="text">
    <string>Credit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
